src/sentry/tasks/post_process.py

```diff
@@ -945,15 +945,20 @@ def process_workflow_engine(job: PostProcessJob) -> None:
 
     from sentry.workflow_engine.processors.workflow import process_workflows
 
-    # PostProcessJob event is optional, WorkflowJob event is required
+    # PostProcessJob event is optional, WorkflowEventData event is required
     if "event" not in job:
-        logger.error("Missing event to create WorkflowJob", extra={"job": job})
+        logger.error("Missing event to create WorkflowEventData", extra={"job": job})
         return
 
     try:
-        workflow_job = WorkflowJob({**job})  # type: ignore[typeddict-item]
+        workflow_job = WorkflowEventData(
```

Comment:
If we're going through this effort, can we also update the variable names?
```suggestion
        workflow_event_data = WorkflowEventData(
```

Reply:
oops